azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -33,6 +33,12 @@ def to_pandas_timedelta(raw_value, timedelta_value) -> "pandas.Timedelta":
     return pd.Timedelta(timedelta_value.total_seconds(), unit="ns")
 
 
+def to_decimal(raw_value, *args):
+    from decimal import Decimal
+
+    return Decimal(raw_value.strip(' "'))
```

Comment:
I would use lstrip & rstrip instead.
Otherwise this conversion may occur [ "1 2" => 12 ]

Reply:
Strip removes characters leading and trailing chars so it will give you the same behavior.

Reply:
Correct.
Sorry for that :)